File: Data Structures & Algorithms/buy-and-sell-crypto/submission-1.cs
public class Solution {
    public int MaxProfit(int[] prices) {
        int res = 0;
        int l = 0, r = 1;
        while(r < prices.Length) {

            //[2,1,2,1,0,1,2]
            //l = 0
            //r = 1
            if(prices[l] > prices[r]) {
                l = r;
            }
            int check = prices[r] - prices[l];
            if(check > res) {
                res = check; 
            }
            r++;
        }
        return res;
    }
}

File: Data Structures & Algorithms/counting-elements/submission-1.js
class Solution {
    /**
     * @param {number[]} arr
     * @return {number}
     */
    countElements(arr) {
        arr.sort();
        var res = 0;
        for(var i = 1; i < arr.length; i++) {
            if((arr[i-1] + 1) === arr[i])
                res++;
        }
        return res;
    }
}
